source/designers-guide/google-pagespeed-best-practise/index.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: default
+shopware_version: 5.3.0
+title: Google PageSpeed Best Practise
+github_link: designers-guide/google-pagespeed-best-practise/index.md
+indexed: true
+group: Frontend Guides
+subgroup: General Resources
+menu_title: Google PageSpeed
+menu_order: 55
+---
+
+<div class="toc-list"></div>
+
+## Introduction
+
+Google PageSpeed is a great tool to get the best performance for your website. It analysis an url and gives some suggestions on how to speed up your website.
```

Comment:
Nope! It is not a great tool.

Suggestion:
"PageSpeed is a tool by Google to indicate the performance of a website. It checks the site for some of the best practice techniques in web development. Google wants to ensure that every site offers a great experience for the user. It is not clear, if the ranking of the tool realy affects the visibility of the site in the Google search algorithm. Although the performance of your website has an affect at the bounce rate and the user experience. So a better ranking in the PageSpeed tool is always a good goal you should work on."